src/Console/Command/CompileCommand.php

```diff
@@ -250,7 +255,7 @@ private function generatePHPBuildCacheKey(InputInterface $input): string
     {
         $c = hash_init('sha256');
 
-        foreach (['os', 'arch', 'php-version'] as $phpBuildOption) {
+        foreach (['os', 'arch', 'php-version', 'php-extensions'] as $phpBuildOption) {
```

Comment:
I change the hash if we don't have the same php extensions, i think it is needed otherwise we could have the same micro sfx even if we changes extensions